src/iasm.c

```diff
@@ -426,14 +426,10 @@ static OPND *asm_and_exp();
 static OPND *asm_cond_exp();
 static opflag_t asm_determine_operand_flags(OPND *popnd);
 static code *asm_genloc(Loc loc, code *c);
-static int asm_getnum();
+static long long asm_getnum();
```

Comment:
Hmmm this is not portable to systems where `sizeof(long long) == sizeof(int)`. Should we worry?